csv.go

```diff
@@ -0,0 +1,157 @@
+package ftdc
+
+import (
+	"context"
+	"encoding/csv"
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/mongodb/mongo-go-driver/bson"
+	"github.com/pkg/errors"
+)
+
+func (c *Chunk) getFieldNames() []string {
```

Comment:
I would argue to put this function and `getRecord` in ftdc.go where Chunk is defined.

Reply:
the methods on the chunk type are pretty distributed closer to call site. I think I agree with you in the big picture about this kind of change, but I'd rather leave it the way it is.

Reply:
sounds good